Document upcoming CI shader builtin MODULATE (#3499)
godotengine/godot-docs: tutorials/shading/shading_reference/canvas_item_shader.rst

+--------------------------------+----------------------------------------------------------------+
| inout vec4 **COLOR**           | Color from vertex primitive.                                   |
+--------------------------------+----------------------------------------------------------------+
| in vec4 **MODULATE**           | Final modulate color.                                          |

> |                                | If used, the default behavior of having the output **COLOR**   |
> |                                | of the fragment function automatically multiplied by it        |
> |                                | won't happen.                                                  |

Reviewer: I think this can be made a little simpler to understand.

```suggestion
|                                | If used, **COLOR** will not be multiplied by modulate          |
|                                | automatically after the fragment function.                     |
```

Author: Much better. Done!
